cdap-docs/examples-manual/source/tutorials/wise.rst

```diff
@@ -51,8 +51,8 @@ In the examples and commands that follow, for brevity we will use these conventi
   on Windows |---| ``bin\cdap-cli.bat``. In the examples given, substitute the actual path
   as appropriate. The CLI allows you to quickly access CDAP facilities from a command line
   environment.
-- The ``curl`` command, common on UNIX-type systems, is included in the CDAP SDK in the
-  ``libexec\bin`` directory as ``curl.exe``.
+- The ``curl`` command, common on UNIX-type systems, is included in a Windows-version in 
```

Comment:
The Windows version of the curl command, common on UNIX-type systems , is included in the CDAP SDK in the